web-registry/src/pages/Description.tsx

```diff
@@ -60,33 +64,35 @@ const Description: React.FC = () => {
           },
         },
       });
-      !isEdit && navigate(`/project-pages/${projectId}/media`);
+      !isEdit && navigateNext();
     } catch (e) {
       // TODO: Should we display the error banner here?
       // https://github.com/regen-network/regen-registry/issues/554
       // console.log(e);
     }
   }
 
+  const Form = (): JSX.Element => (
+    <DescriptionForm
+      submit={submit}
+      onNext={navigateNext}
+      onPrev={() => navigate(`/project-pages/${projectId}/roles`)}
```

Comment:
Should this also be declared above like navigateNext for consistency ?

Reply:
@flagrede similar to my above comment, I tend to only extract simple functions like this when they're reused elsewhere. `navigateNext` is used both here, and in the `submit` function. Again though, happy to adjust if folks prefer a particular pattern

Reply:
@flagrede just extracted them all to separate handler functions 👍 see: https://github.com/regen-network/regen-web/pull/989/commits/0a7dc31db04b50e17f90d9e9096181cb2b7518e5